erase-install.sh

```diff
@@ -1635,6 +1649,11 @@ while test $# -gt 0 ; do
             shift
             account_shortname="$1"
             ;;
+        --max-password-attempts)
+            shift
+            max_password_attempts="$1"
```

Comment:
What happens if somebody adds `--max-password-attempts 0`?

Reply:
This is a quite valid point, then undefined behaviour would ensue. I've added more strict checking of the parameter so it should contain only valid values.
The password attempts are now logged as well.